src/lib/components/QuestionCard.svelte

```
@@ -40,57 +40,83 @@
-			selectedQuestions.push({
-				question_revision_id: questionId,
-				response: [response],
-				visited: true,
-				time_spent: 0
-			});
 		}
 		updateStore();
+
+		try {
+			await submitAnswer(questionId, newResponse);
+		} catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)